react/features/toolbox/components/web/Toolbox.tsx

```diff
@@ -845,6 +846,12 @@ class Toolbox extends Component<Props> {
         };
 
 
+        const whiteboard = {
```

Comment:
Shouldn't we check if we can show the button here?

Reply:
I thought it would look better if we had the check in the props (visible), doesn't it work the same?

Reply:
yeah that works